### ISSUE TITLE
Lua Error Ele Shaman

### ISSUE THREAD

**Issue by Morl0ck** [mwojtkowski/Ovale#25]
So far this happens when i put down a capacitor totem in a BG or arena.

```
Message: Interface\AddOns\Ovale\dist\Power.lua:393: attempt to index local 'powerInfo' (a nil value)
Time: Wed Aug  8 19:21:54 2018
Count: 1159
Stack: Interface\AddOns\Ovale\dist\Power.lua:393: attempt to index local 'powerInfo' (a nil value)
Interface\AddOns\Ovale\dist\Power.lua:393: in function 'UpdatePower'
Interface\AddOns\Ovale\dist\Power.lua:356: in function '?'
...nIn\libs\CallbackHandler-1.0\CallbackHandler-1.0.lua:145: in function <...nIn\libs\CallbackHandler-1.0\CallbackHandler-1.0.lua:145>
[string "safecall Dispatcher[3]"]:4: in function <[string "safecall Dispatcher[3]"]:4>
[C]: ?
[string "safecall Dispatcher[3]"]:13: in function '?'
...nIn\libs\CallbackHandler-1.0\CallbackHandler-1.0.lua:90: in function `Fire'
...AddOns\Bartender4\libs\AceEvent-3.0\AceEvent-3.0.lua:120: in function <...AddOns\Bartender4\libs\AceEvent-3.0\AceEvent-3.0.lua:119>

Locals: <none>
```

**Comment by Morl0ck** [comment]
Looks like the `powerType` when i drop the capacitor totem is `mana`

**Comment by mwojtkowski** [comment]
It is same problem as #23 and Sidoine/Ovale#461 

it should be fixed with pull request Sidoine/Ovale#465 merge ( pending )
